File: JAVA/lcmnumber.java
import java.util.Scanner;

public class lcmnumber {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter first number :");
        int num1 = sc.nextInt();

        System.out.println("Enter second number :");
        int num2 = sc.nextInt();

        int max = Math.max(num1, num2);
        int lcm = max;

        while (true) {
            if (lcm % num1 == 0 && lcm % num2 == 0) {
                break;
            }
            lcm++;
        }
        System.out.println("LCM of " + num1 + " and " + num2 + " is " + lcm);
    }
}

File: BY-PYTHON/LCMnumber.py
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))

for i in range(max(num1, num2), (num1 * num2) + 1):
    if i % num1 == 0 and i % num2 == 0:
        lcm = i
        break

print(f"LCM of {num1} and {num2} is {lcm}")
